build.gradle

```diff
@@ -103,6 +103,9 @@ allprojects {
 
   repositories {
     mavenCentral()
+    maven {
```

Comment:
TODO to remove it when kotlinpoet ships? or is it really a problem to have it always? I don't really know.

Reply:
You cannot release with a snapshot dependency, yes

Reply:
Sure, KotlinPoet 1.1 goes live soon